config/checkstyle-examples-suppressions.xml

```diff
@@ -6,6 +6,12 @@
 
 <suppressions>
 
+  <!-- These files have no class in java file by design. -->
```

Comment:
not related to this line.

Please update description of Check from `Checks whether file contains code. Files which are considered to have no code: ` to `Checks whether file contains code. Java compiler is not raising errors on java files that has no code or all is commentend out. Files which are considered to have no code: `

Reply:
I have updated `NoCodeInFileCheck.java`
Should I apply the same for [InputJavadocMetadataScraperNoCodeInFileCheck.java](https://github.com/checkstyle/checkstyle/blob/master/src/test/resources/com/puppycrawl/tools/checkstyle/meta/javadocmetadatascraper/InputJavadocMetadataScraperNoCodeInFileCheck.java) also ?